src/utils/index.js

```diff
@@ -2,10 +2,10 @@ import axios from 'axios';
 import jwtDecode from 'jwt-decode';
 
 export const getUrl = (hostName) => {
-  if (hostName.includes('master')) {
-    return 'https://indiana-ah-master.herokuapp.com/api/v1/';
+  if ((hostName.includes('staging')) || (hostName.includes('localhost'))) {
+    return 'https://indiana-ah-staging.herokuapp.com/api/v1/';
   }
-  return 'https://indiana-ah-staging.herokuapp.com/api/v1/';
+  return 'https://indiana-ah-master.herokuapp.com/api/v1/';
```

Comment:
What was wrong with the previous logic?

Reply:
The previous logic was checking if the hostname includes `master` and then pointing it to the master backend deployment. This failed because our master front-end deployment(indiana-ah-frontend.herokuapp.com) doesn't have `master` in its hostname.